lib/travis/build/script/julia.rb

```diff
@@ -108,8 +104,7 @@ def julia_url
             end
             case julia_version = Array(config[:julia]).first.to_s
             when 'release'
-              # CHANGEME on new minor releases (once or twice a year)
-              url = "julialang-s3.julialang.org/bin/#{osarch}/0.6/julia-0.6-latest-#{ext}"
+              sh.failure "'release' is no longer a valid julia version specifier in .travis.yml"
             when 'nightly'
               url = "julialangnightlies-s3.julialang.org/bin/#{osarch}/julia-latest-#{nightlyext}"
             when /^(\d+\.\d+)\.\d+$/
```

Comment:
oh I was wrong you'd need to also add a
```
when /^(\d+)$/
  url = "julialang-s3.julialang.org/bin/#{osarch}/#{$1}/julia-#{$1}-latest-#{ext}"
```
to the switch-case below to make just specifying the major ver possible, in addition to needing the corresponding file to exist on s3

Reply:
I'll wait until it is properly supported.

We will probably want 1.0 nightlies at some point as well.